frontends/web/src/locales/en/app.json

```diff
@@ -501,18 +501,21 @@
       "deviceName": {
         "description": "Change the name of your device."
       },
+      "firmware": {
+        "upToDate": "Your firmware is up to date."
+      },
       "securechip": {
         "description": "The model of the secure chip."
       },
       "title": "Device information"
     },
     "expert": {
       "factoryReset": {
-        "description": "Reset your device to factory settings. This deletes your wallet!",
+        "description": "Reset your device to factory settings. This deletes the wallet on your BitBox02!",
         "title": "Factory reset"
       },
       "goToStartupSettings": {
-        "description": "Enters the bootloader of the BitBox02. Can enable firmware hash from here."
+        "description": "Enter the bootloader of the BitBox02. You can enable the firmware the hash from here."
```

Comment:
As mentioned in the PR's description, since these keys are not "live" on locize yet it should be ok to edit it here.


... right..? 😄

Reply:
correct.